knowledge_base/errors/nice2_build.rst

```diff
@@ -0,0 +1,115 @@
+
+        [INFO] -------------------------------------------------------------
+        [ERROR] COMPILATION ERROR :
+        [INFO] -------------------------------------------------------------
+        [ERROR] error reading /home/tocco/.m2/repository/ch/tocco/manager/tocco-manager-alerts/1.2-SNAPSHOT/**tocco-manager-alerts**-1.2-SNAPSHOT.jar; **error in opening zip file**
+        [ERROR] error reading /home/tocco/.m2/repository/ch/tocco/manager/tocco-manager-alerts/1.2-SNAPSHOT/**tocco-manager-alerts**-1.2-SNAPSHOT.jar; **error in opening zip file**
+        [ERROR] /build/agent/work/74cb698015a78e5e/monitoring/api/src/main/java/ch/tocco/nice2/monitoring/AlertPublisher.java:[3,31] package ch.tocco.manager.alerts does not exist
+        [ERROR] /build/agent/work/74cb698015a78e5e/monitoring/api/src/main/java/ch/tocco/nice2/monitoring/AlertPublisher.java:[11,25] cannot find symbol
+          symbol:   class Alert
+          location: interface ch.tocco.nice2.monitoring.AlertPublisher
+        [ERROR] /build/agent/work/74cb698015a78e5e/monitoring/api/src/main/java/ch/tocco/nice2/monitoring/AlertPublisher.java:[12,26] cannot find symbol
+          symbol:   class Alert
+          location: interface ch.tocco.nice2.monitoring.AlertPublisher
+
+
+Error :green:`B`: Corrupted Pom
+```````````````````````````````
+
+Cause:
+
+    .. code::
+
+        An invalid XML character (Unicode: 0xb) was found in the value of attribute "title" and element is "article".
+
+Full message:
+
+.. parsed-literal::
+
+    [Step 7/10] [INFO] ------------------------------------------------------------------------
+    [Step 7/10] [INFO] Building nice2-optional-apprenticeshipcorrespondence-module 1.0-SNAPSHOT
+    [Step 7/10] [INFO] ------------------------------------------------------------------------
+    [Step 7/10] Importing data from '/build/agent/work/59745ac4296dbd52/optional/apprenticeshipcorrespondence/module/target/failsafe-reports/TEST-\*.xml' (not existing file) with 'surefire' processor
+            [ch.tocco.nice2.optional.apprenticeshipcorrespondence:nice2-optional-apprenticeshipcorrespondence-module] **[Fatal Error] :1:25: An invalid XML character (Unicode: 0xb) was found in the value of attribute "title" and element is "article".**
+            [ch.tocco.nice2.optional.apprenticeshipcorrespondence:nice2-optional-apprenticeshipcorrespondence-module] ##teamcity[importData tc:tags='tc:internal' type='surefire' path='/build/agent/work/59745ac4296dbd52/optional/apprenticeshipcorrespondence/module/target/surefire-reports/TEST-\*.xml' whenNoDataPublished='nothing' logAsInternal='true']
+
+
+Cause
+`````
+
+In both cases, :blue:`A` and :green:`B`, a file received from :term:`Artifactory` is corrupted. In case of a ``*.jar``,
+the resulting error is en extraction error (:blue:`A`) and in case of a ``pom.xml`` a parser error (:green:`B`).
+
+The corruption is usually caused by and :term:`Remote Repository` used by our :term:`Artifactory`. The latest incident
```

Comment:
word `and` should be removed? (... is usually caused by AND :term:`Remote Repository` used...)